packages/cli/test/integration.js

```diff
@@ -1088,30 +1187,6 @@ test('output the version', async t => {
   t.is(version, pkg.version);
 });
 
-test('should error with suggestion for secrets subcommand', async t => {
```

Comment:
Why is this test being removed?

Reply:
The suggestion logic for alternate singular/plural was removed. It was only used when there was a conflict between a subdirectory and a command. We have a better way to resolve that now, which is to use the flag if you meant to deploy or let the command run, which is the new behavior.

I thought about moving this logic to the failure cause where the first argument was neither a command or a subdirectory, but this logic was very narrow. It checked plural/singular variants by adding or removing a trailing `s`, which would miss things like `alias` -> `aliases` because it would check `aliass`.

Reply:
That reminds me: the `subdirectory-secret` fixture is now no longer used. I'll delete that too.